### DISCUSSION TITLE
Traffic related many queries / Insight

### DISCUSSION THREAD

**Discussion by AnimishY** [community/community#22090] [Other Feature Feedback, Questions, & Ideas]
<p>I created a website in HTML, and published it using GitHub Pages.</p>
<p>Now i regularly go to see traffic on my repository, and I have multiple questions about it.</p>
<ol>
<li>Can I use the views under traffic as count of people viewed my repository or the website?</li>
<li>My visit to my repository also count as a visit?</li>
<li>What’s the difference between unique visitors and visitors?</li>
<li>Who are cloners?</li>
<li>What is the difference between unique cloners and cloners?</li>
<li>Is there any way to know how many visitors visited my website?</li>
</ol>
<p>Thanks!!</p>

**Comment by otter-computer** [comment, marked as answer]
<blockquote>
<ol>
<li>Can I use the views under traffic as count of people viewed my repository or the website?</li>
</ol>
</blockquote>
<p>The traffic insights inside your repository only relate to view of your repository on <a href="http://GitHub.com">GitHub.com</a>, not views of your Pages site.</p>
<blockquote>
<ol start="2">
<li>My visit to my repository also count as a visit?</li>
</ol>
</blockquote>
<p>No, viewing your own repository while signed in doesn’t count towards this data.</p>
<blockquote>
<ol start="3">
<li>What’s the difference between unique visitors and visitors?</li>
</ol>
</blockquote>
<p>Unique visitors are counts how many different people have visited your repository page.</p>
<p>If your regular visitors count is higher than your unique visitors count this means that some people have visited this repository on more than one occasion.</p>
<blockquote>
<ol start="4">
<li>Who are cloners?</li>
</ol>
</blockquote>
<p>Cloners are people that have cloned (downloaded) your site’s code to their computer.</p>
<blockquote>
<ol start="5">
<li>What is the difference between unique cloners and cloners?</li>
</ol>
</blockquote>
<p>Similar to visitors and unique visitors, if cloners is higher than unique cloners this means that some people have cloned your repository more than once.</p>
<blockquote>
<ol start="6">
<li>Is there any way to know how many visitors visited my website?</li>
</ol>
</blockquote>
<p>GitHub doesn’t track visits to Pages sites. To track this data you’ll need to use a 3rd party tool like Google Analytics.</p>

**Comment by AnimishY** [comment]
<p>Hey, Thanks for reply, Concepts more clear now!!! :sunglasses:</p>
<p><a class="mention" href="/thomasshaped">@thomasshaped</a></p>

**Comment by yashu1wwww** [comment]
my github profile visits count decreased to 40k+ to 200 what nonsense is these github?
